<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sd_t_Form</class>
 <widget class="QWidget" name="Sd_t_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>t分布</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>424</x>
     <y>19</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>t分布图像：</string>
   </property>
  </widget>
  <widget class="Plot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>431</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>153</y>
     <width>287</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>输入n值：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>显示t分布图像</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>输入α值</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QDoubleSpinBox" name="doubleSpinBox_t_arfa">
      <property name="minimum">
       <double>0.010000000000000</double>
      </property>
      <property name="maximum">
       <double>0.990000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="pushButton_t_quantile_plot">
      <property name="text">
       <string>显示分位点图像</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="pushButton_t_quantile">
      <property name="text">
       <string>显示分位点</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QLineEdit" name="lineEdit_t_quantile">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_one">
      <property name="text">
       <string>单个图像</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_few">
      <property name="text">
       <string>多个图像</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Plot</class>
   <extends>QWidget</extends>
   <header>Plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
